ai_economist/foundation/private_key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Clarify name and move into COVID subdir. This looks like a leak right now. FYI: It's not!

Reply:
moved!